ci/cpu/prebuild.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+export BUILD_ABI=1
+export BUILD_CUDF=1
+export BUILD_CFFI=1
+
+if [[ "$PYTHON" == "3.6" ]]; then
+    export BUILD_LIBCUDF=1
+else
+    export BUILD_LIBCUDF=0
```

Comment:
Is there a reason we're only building libcudf on python 3.6?

Reply:
This is a consequence of our Matrix: Python 3.6 & 3.7, CUDA 9.2, CUDA 10.0. By only building libcudf on Python 3.6, we end up only building it once per CUDA version.

Reply:
👍 sounds good.